.github/workflows/tests.yml

```diff
@@ -175,62 +203,6 @@ jobs:
           name: css
           path: evap/static/css/evap.css
 
-  render_pages:
-    runs-on: ubuntu-22.04
-
-    name: Render Html pages
-
-    steps:
-      - uses: actions/checkout@v4
-      - uses: ./.github/setup_evap
-        with:
-          start-db: true
-
-      - name: Render pages
-        run: coverage run manage.py ts render_pages
-      - name: Upload coverage
-        uses: codecov/codecov-action@v5
-        with:
-          flags: render-pages
-          token: ${{ secrets.CODECOV_TOKEN }}
-      - name: Store rendered pages
-        uses: actions/upload-artifact@v4
-        with:
-          name: rendered-pages
-          path: evap/static/ts/rendered
-
-  typescript:
```

Comment:
Don't we still have unit tests in Typescript?

Reply:
done